charts/trino/README.md.gotmpl

```diff
@@ -0,0 +1,25 @@
+{{ template "chart.header" . }}
```

Comment:
I think I should add a `.helmignore` file so this template is not packaged in the chart, or put it in a different location and point to it when calling `helm-doc` from `.pre-commit-config.yaml`.

Reply:
That sounds good too .. you can do that in this or a separate PR.